packages/react-server-cli/bin/react-server-cli

```diff
@@ -1,8 +1,5 @@
 #!/usr/bin/env node
 
 require("babel-core/register");
-require.extensions['.css'] = function(module, filename) {
-  return module._compile("", filename);
-};
```

Comment:
In order for require("foo.css") to work on server-side in tests, I had to move this to `index.js`.